package.json

```diff
@@ -56,6 +56,9 @@
     "docker:scripts:docker:build:local": "yarn docker:scripts:docker:build --npm-registry local --npm-tag local --docker-tag local",
     "docker:scripts:docker:publish": "docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -e DOCKERHUB_USERNAME -e DOCKERHUB_TOKEN api3/airnode-packaging:latest docker publish",
     "docker:scripts:docker:publish:latest": "yarn docker:scripts:docker:publish",
+    "docker:scripts:github": "docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -e GITHUB_TOKEN api3/airnode-packaging:latest github",
```

Comment:
You need to set the GH token when running this command. It might be nice to mention this somewhere. Maybe root README?

Reply:
There's a README in the `docker` directory that I'm forgetting to update. I'll do that.